pkg/image/airgap.go

```diff
@@ -187,6 +191,105 @@ func TagAndPushImagesFromBundle(airgapBundle string, options imagetypes.PushImag
 	return nil
 }
 
+func PathToRegistryECImage(srcImage, registryNamespace string) string {
+	imageParts := strings.Split(srcImage, "/")
+	imageName := imageParts[len(imageParts)-1]
+	return filepath.Join(registryNamespace, "embedded-cluster", imageName)
+}
+
+func PushECImagesFromTempRegistry(airgapRootDir string, airgap *kotsv1beta1.Airgap, options imagetypes.PushImagesOptions) error {
```

Comment:
How much effort would jt be to add a unit test for this function similar to the test we have for pushing embedded cluster artifacts?

Reply:
That would consume me half a day, at the very least. The test is much more closer to PushImagesFromTempRegistry than to the artifacts function. Let me know if I should start it.

Reply:
No need, we can cover it in the e2e test. Thank you.